googleMaps.js

```diff
@@ -1,46 +1,92 @@
-// googleMaps.js
-const { Client } = require("@googlemaps/google-maps-services-js");
-const dotenv = require("dotenv");
+const fetch = require("node-fetch");
+require("dotenv").config();
 
-dotenv.config();
+const API_KEY = process.env.GOOGLE_MAPS_API_KEY;
 
-const client = new Client({});
+// Function to get coordinates from an address
+async function getCoordinates(address) {
+  try {
+    const response = await fetch(
+      `https://maps.googleapis.com/maps/api/geocode/json?address=${encodeURIComponent(
```

Comment:
Suggestion: Store the first part of this URL that is reused in other fetch requests below in a `BASE_URL` variable